.ci/release.sh

```diff
@@ -43,7 +43,7 @@ operator-sdk olm-catalog gen-csv \
     --from-version=${PREVIOUS_VERSION}
 
 # changes to deploy/olm-catalog/jaeger-operator/newversion/...
-sed "s~${PREVIOUS_VERSION}~${OPERATOR_VERSION}~gi" -i deploy/olm-catalog/jaeger-operator/${OPERATOR_VERSION}/jaeger-operator.v${OPERATOR_VERSION}.clusterserviceversion.yaml
+sed "s~containerImage: docker.io/jaegertracing/jaeger-operator:${PREVIOUS_VERSION}~containerImage: docker.io/jaegertracing/jaeger-operator:${OPERATOR_VERSION}~i" -i deploy/olm-catalog/jaeger-operator/${OPERATOR_VERSION}/jaeger-operator.v${OPERATOR_VERSION}.clusterserviceversion.yaml
```

Comment:
what about `image: jaegertracing/jaeger-operator:1.15.1` in the spec - this wouldn't change the version here. Possibly just make it substitute from `jaeger-operator:....` ?

Reply:
That one is changed by the `gen-csv` command.